tests/src/end-to-end/linktoolbar/linktoolbar.test.ts

```
@@ -0,0 +1,50 @@
+    await page.keyboard.press("Shift+Home");
+    await page.waitForSelector(LINK_BUTTON_SELECTOR);
+    await page.click(LINK_BUTTON_SELECTOR);
+    await page.keyboard.type("https://example.com");
+    await page.keyboard.press("Enter");
+
+    // Move cursor back onto the linked text to trigger link toolbar
+    await page.keyboard.press("ArrowLeft");
+    await page.waitForTimeout(500);
+
+    // Click Edit link button
+    const editButton = page.getByText("Edit link");
```

ESLint

[error] Avoid destructuring queries from `render` result, use `screen.getByText` instead

(testing-library/prefer-screen-queries)